graphula-core/src/Graphula.hs

```diff
@@ -554,3 +561,13 @@ newtype Only a = Only { fromOnly :: a }
 
 only :: a -> Only a
 only = Only
+
+generate :: (MonadIO m, MonadGraphulaBackend m) => Gen a -> m a
```

Comment:
I'm thinking about renaming this and exposing it. That way anything in tests that utilizes random generation can use the `graphula` seed and remain deterministic, instead of falling back to `IO`.

Reply:
I kind of wish we could export it at this name, to make it clear(er) that you _should_ do that to keep it all deterministic within the test. But I also agree with the fear of import churn to deal with sudden naming conflicts in any existing tests also using `Arbitrary`.

What if we exported it as `generate`, but from a new module named `Graphula.Arbitrary`? Not super discoverable, but at least a little clearer (IMO) that, if you want to do `Arbitrary` stuff in a `Graphula` test, you should use these (conflicting) names from that module instead?